R/restrict_derivation.R

```diff
@@ -6,6 +6,11 @@
 #'
 #' @param derivation Derivation
 #'
+#'   A function is expected. The function must provide the `dataset` argument
+#'   and all arguments specified in the `params()` objects passed to the `arg`
+#'   argument. Therefore it is not possible to specify dplyr functions like
+#'   `mutate()` or `summarize()`.
```

Comment:
```suggestion
#'   An `{admiral}` function that performs a specific derivation is expected. The function must 
# '  provide the `dataset` argument
#'   and all arguments specified in the `params()` objects passed to the `arg`
#'   argument. Please note, it is not possible to specify `{dplyr}` functions like
#'   `mutate()` or `summarize()`.
```

Reply:
Possible suggestion?

Reply:
I would not restrict to "admiral" because a derivation could be implemented in TA, company, or study specific admiral packages.